inst/htmlwidgets/plugins/Leaflet-MiniMap/Minimap-binding.js

```diff
@@ -1,11 +1,24 @@
-  LeafletWidget.methods.addMiniMap = function(position, width, height , collapsedWidth,
```

Comment:
We are not going from 1.0.2 to 1.0.3, right ? So just add the new stuff under existing 1.0.2 ?